backend/server.py

```
@@ -378,28 +393,40 @@ def _get_chat_or_404(
+            if tool_call.get("name") != "vectordb_search":
+                continue
+
+            call_id = tool_call.get("id")
 
-    return False
+            for followup in messages[i + 1 :]:
+                if isinstance(followup, ToolMessage) and followup.tool_call_id == call_id:
+                    try:
+                        return json.loads(followup.content)
+                    
+                    except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)